indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -152,6 +152,15 @@
     {% endif %}
 
     <div class="toolbar f-j-end hide-if-locked">
+        <div class="group {% if email_tpls %} hidden {% endif %}">
+            <button class="i-button"
+                    data-href="{{ url_for('.quick_setup_notifications', event) }}"
+                    data-title="{% trans %}Quick setup{% endtrans %}"
+                    data-update="#email-template-manager"
+                    data-method="POST">
+            {% trans %}Quick setup{% endtrans %}
```

Comment:
What about making this part of the notification instead of a button nested inside this dialog? That way you also avoid the display issue of the jumping dialog since it won't be open when clicking the button

Reply:
I thought about it, but I thought it might add too much noise to the managing section, whilst adding it to the notifications form would feel like a part of it. I would keep it as it is, but if you have a strong opinion, I don't mind, specially after hiding it if already configured (what I didn't think when I was deciding where to place it)

Reply:
> What about making this part of the notification instead of a button nested inside this dialog?

+1, this seems more obvious for the user. Should clicking the button open the notification settings afterwards?

Reply:
I'd just create them and flash a message telling the user that they can customize them. I think clicking the correct button to open the dialog isn't too hard for the user :p

Reply:
> I think clicking the correct button to open the dialog isn't too hard for the user :p

famous last words :sweat_smile:

Reply:
Perfect, I have already applied the other changes in my local, I make this one and upload all the changes ok? hope to merge it by today or tomorrow if possible please

Reply:
I have done it but I am not convinced at all about how does it work. The reason is you can create some notifications and then use the quick setup. I would either not hide the button or return it to it's place, what do you think about it?

Reply:
IMHO it's fine. It covers the case where people do not realize they need to set up notifications. They click the button and have a good starting point. I think users who started setting up notifications are aware that they need to do it...

Reply:
![image](https://github.com/user-attachments/assets/15708330-70b6-46b3-8c17-0f55693a840d)

However, the part I do not really like is that the button and message are separated. Why not include the button within that message instead of having it separately below?